docs/requirements.txt

```diff
@@ -1,3 +1,3 @@
 sphinx-readable-theme==1.3.0
 aiohttp==3.7.4
```

Comment:
¯\_(ツ)_/¯ I was aiming for the minimal set of dependencies to update. I only added the `mypy` because it wouldn't build locally on `arm64` without updating it. And in any case, `docs/requirements.txt` is only used for building the docs so I really don't see a huge reason.

If you think the `aiohttp` dependency needs to be updated, please feel free to open a pull request doing so.